catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/TransformersCommand.java

```diff
@@ -0,0 +1,141 @@
+
+    private static final String MIME_TYPE = "mime-type";
+
+    private static final String SCHEMA = "schema";
+
+    private static final String NOT_AVAILABLE = "N/A";
+
+    private static final String LINE = "------------------------";
+
+    @Override
+    protected Object executeWithSubject() throws Exception {
+
+        // ServiceReferences are needed to get property information from InputTransformers
+        List<ServiceReference<InputTransformer>> serviceReferences =
+                bundleContext.getServiceReferences(InputTransformer.class, null)
+                        .stream()
+                        .filter(ref -> ref.getProperty(Constants.SERVICE_ID) != null)
+                        .collect(Collectors.toList());
+
+        List<TransformerProperties> transformersProperties = serviceReferences.stream()
+                .map(TransformerProperties::new)
+                .collect(Collectors.toList());
+
+        int activeTransformers = transformersProperties.size();
+
+        if (activeTransformers == 0) {
+            console.printf("There are no active transformers%n%n");
+
+        } else {
+            console.printf("%nActive Transformers: %d%n%s%n%n", activeTransformers, LINE);
+
+            Iterator<TransformerProperties> tpIterator = transformersProperties.iterator();
+            TransformerProperties tp;
+
+            while (tpIterator.hasNext()) {
+                tp = tpIterator.next();
+                console.printf("%s", tp.printProperties());
+
+                if (tpIterator.hasNext()) {
+                    console.printf("%n%s%n", StringUtils.repeat(LINE, 4));
+                }
+                console.printf("%n");
+            }
+        }
+
+        return null;
+    }
+
+    private static class TransformerProperties {
+
+        private String id;
+
+        private String schema;
+
+        private List<String> mimeTypes;
+
+        public TransformerProperties(ServiceReference ref) {
+            this.id = getTransformerProperty(ref, Constants.SERVICE_ID);
+            this.schema = getTransformerProperty(ref, SCHEMA);
+            this.mimeTypes = getTransformerMimeProperty(ref);
+        }
+
+        public String printProperties() {
+            StringBuilder s = new StringBuilder(MessageFormat.format(
+                    "{0}: {1}\n\n\t{2}: {3}\n\t{4}s: {5}\n",
+                    Constants.SERVICE_ID,
+                    id,
+                    SCHEMA,
+                    schema,
+                    MIME_TYPE,
+                    mimeTypes.remove(0)));
+
+            for (String mimeType : mimeTypes) {
+                s.append("\t\t    ")
+                        .append(mimeType)
+                        .append("\n");
+            }
+            return s.toString();
+        }
+
+        private String getTransformerProperty(ServiceReference ref, String property) {
+            return Optional.ofNullable(ref.getProperty(property))
+                    .map(String.class::cast)
```

Comment:
✏️ I know you've applied the suggestion I made in an earlier review and I do think it's sufficient here, but generally, you'll want to filter prior to the cast to prevent class cast exceptions with
```java
                    .filter(String.class::isInstance)
```
It's probably best to make the change now lest someone see this in the future and use it as a model without understanding that the instance check was intentionally elided.